src/main/java/ru/mercuriev/Controller.java

```diff
@@ -17,7 +18,7 @@
     @RequestMapping(value = "/next", method = RequestMethod.POST)
     @ResponseBody
     public int[][] next(@RequestBody int[][] gen) throws IOException {
```

Comment:
The idea was to move classes responsible for GoF implementation to the separated package